staging/src/k8s.io/apimachinery/pkg/runtime/swagger_doc_generator.go

```diff
@@ -60,24 +61,40 @@ func fmtRawDoc(rawDoc string) string {
 		}
 	}
 
-	// Ignore all lines after ---
-	rawDoc = strings.Split(rawDoc, "---")[0]
+	var isPrevLinePartOfListOrTable bool
+	isPrevLinePartOfListOrTable = false
```

Comment:
This value is only ever changed in the `default` case below. Is that by intention? I would have expected it to be reset to `false` each iteration.

Reply:
After seeing your comment, I realized that the assignment `isPrevLinePartOfListOrTable = false` is required to all the cases other than `default`.
But in this implementation, `isPrevLinePartOfListOrTable` should not be reset to `false` each iteration,
because the purpose of this variable is to save the last iteration's status.
PTAL and please let me know if I am wrong.

Reply:
Yes I was expecting to see some way for the non-default cases to reset the flag